site/src/api/api.ts

```diff
@@ -1277,9 +1277,14 @@ class ApiMethods {
 
 	cancelWorkspaceBuild = async (
 		workspaceBuildId: TypesGen.WorkspaceBuild["id"],
+		request?: TypesGen.CancelWorkspaceBuildParams,
```

Comment:
Since the type has the "Params" sufix, I would name this variable as `params` instead of `request`.